docs/examples/custom_operations/gpu_python_operator.ipynb

```diff
@@ -69,6 +69,21 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "⚠️**Warning**⚠️\n",
```

Comment:
What am I looking at here? Some special characters, or is it github doing strange things? Is it even intentional?
It looks like this:
![image](https://github.com/NVIDIA/DALI/assets/17044837/35aafc21-5e6d-4f9f-858d-aa5357e717b4)

Reply:
It is intentional. It renders as 
![image](https://github.com/NVIDIA/DALI/assets/39967756/cdbfd8be-726e-4f7d-97a5-7673ae5b9bfc)
inside our docs. I can remove it you have strong feelings about it.